trillian/integration/hammer.go

```diff
@@ -449,42 +451,51 @@ func (s *hammerState) oneOp(ctx context.Context) error {
 	ep := s.cfg.EPBias.Choose()
 	glog.V(3).Infof("perform %s operation", ep)
 	status := http.StatusOK
+	defer func() {
+		s.stats.done(ep, status)
+		s.totalOps++
```

Comment:
So retries count as a single op?  What operations have you seen succeed after a retry?